ggml.c

```diff
@@ -289,8 +287,8 @@ inline static void * ggml_calloc(size_t num, size_t size) {
 #endif
 #elif defined(GGML_USE_CLBLAST)
 #include "ggml-opencl.h"
-#elif defined(GGML_USE_VULKAN)
-#include "ggml-vulkan.h"
+#elif defined(GGML_USE_SYCL)
+#include "ggml-sycl.h"
```

Comment:
This is probably an artifact from the merge. It should be removed, `ggml-sycl.h` does not to be included in `ggml.c` anymore.

Reply:
Thanks. I used a new tool for the merge and I guess I didn't understand correctly what it was trying to tell me.